src/snark/libsnark/relations/arithmetic_programs/qap/tests/test_qap.cpp

```diff
@@ -86,6 +86,8 @@ void test_qap(const size_t qap_degree, const size_t num_inputs, const bool binar
 
 TEST(relations, qap)
 {
+    alt_bn128_pp::init_public_params();
```

Comment:
I'm confused because `src/snark` has been removed. Why didn't this cause a conflict on the rebase?

Reply:
Oh because it was a merge of master into the branch, not a rebase. (I find that a rebase is usually preferable for old PRs.)

Reply:
This file was removed in upstream/master, so it has been removed via the merge commit.